docs/multicluster/setup-multi-primary.sh

```diff
@@ -0,0 +1,216 @@
+kubectl config use-context "${CTX_CLUSTER2}"
+kubectl get ns sail-operator --context "${CTX_CLUSTER2}" || make -C "${SCRIPT_DIR}/../.." deploy
+
+# 3. Create istio-system namespace on each cluster and configure a common root CA. 
+
+kubectl get ns istio-system --context "${CTX_CLUSTER1}" || kubectl create namespace istio-system --context "${CTX_CLUSTER1}"
+kubectl --context "${CTX_CLUSTER1}" label namespace istio-system topology.istio.io/network=network1
+kubectl get secret -n istio-system --context "${CTX_CLUSTER1}" cacerts || kubectl create secret generic cacerts -n istio-system --context "${CTX_CLUSTER1}" \
+  --from-file=${CERTS_DIR}/east/ca-cert.pem \
+  --from-file=${CERTS_DIR}/east/ca-key.pem \
+  --from-file=${CERTS_DIR}/east/root-cert.pem \
+  --from-file=${CERTS_DIR}/east/cert-chain.pem
+
+kubectl get ns istio-system --context "${CTX_CLUSTER2}" || kubectl create namespace istio-system --context "${CTX_CLUSTER2}"
+kubectl --context "${CTX_CLUSTER2}" label namespace istio-system topology.istio.io/network=network2
+kubectl get secret -n istio-system --context "${CTX_CLUSTER2}" cacerts || kubectl create secret generic cacerts -n istio-system --context "${CTX_CLUSTER2}" \
+  --from-file=${CERTS_DIR}/west/ca-cert.pem \
+  --from-file=${CERTS_DIR}/west/ca-key.pem \
+  --from-file=${CERTS_DIR}/west/root-cert.pem \
+  --from-file=${CERTS_DIR}/west/cert-chain.pem
+
+# 4. Create Sail CR on east
+
+kubectl apply --context "${CTX_CLUSTER1}" -f - <<EOF
+apiVersion: operator.istio.io/v1alpha1
+kind: Istio
+metadata:
+  name: default
+spec:
+  version: ${SAIL_VERSION}
+  namespace: istio-system
+  values:
+    pilot:
+      resources:
+        requests:
+          cpu: 100m
+          memory: 1024Mi
+    global:
+      meshID: mesh1
+      multiCluster:
+        clusterName: east
+      network: network1
+EOF
+
+kubectl wait --context "${CTX_CLUSTER1}" --for=jsonpath='{.status.revisions.ready}'=1 istios/default --timeout=3m
+
+# 5. Create east-west gateway on east
+curl -fsL "https://raw.githubusercontent.com/istio/istio/${ISTIO_VERSION}/samples/multicluster/gen-eastwest-gateway.sh" | \
+  bash -s -- --mesh mesh1 --cluster east --network network1 | \
+  istioctl manifest generate -f - | \
+  kubectl apply --context "${CTX_CLUSTER1}" -f -
+
+# 6. Expose services on east
+kubectl --context "${CTX_CLUSTER1}" apply -n istio-system -f "https://raw.githubusercontent.com/istio/istio/${ISTIO_VERSION}/samples/multicluster/expose-services.yaml"
+
+# 7. Create Sail CR on west
+
+kubectl apply --context "${CTX_CLUSTER2}" -f - <<EOF
+apiVersion: operator.istio.io/v1alpha1
+kind: Istio
+metadata:
+  name: default
+spec:
+  version: ${SAIL_VERSION}
+  namespace: istio-system
+  values:
+    pilot:
+      resources:
+        requests:
+          cpu: 100m
+          memory: 1024Mi
+    global:
+      meshID: mesh1
+      multiCluster:
+        clusterName: west
+      network: network2
+EOF
+
```

Comment:
Same here- we should wait for the operator to be ready before attempting to read resource status

```suggestion

kubectl wait --context "${CTX_CLUSTER2}" --for=condition=Available deployment/sail-operator -n sail-operator --timeout=3m
```